pkg/kubectl/cmd/util/factory.go

```diff
@@ -110,6 +110,8 @@ type ClientAccessFactory interface {
 	FederationClientForVersion(version *schema.GroupVersion) (*restclient.RESTClient, error)
 	// TODO remove.  This should be rolled into `ClientSet`
 	ClientSetForVersion(requiredVersion *schema.GroupVersion) (internalclientset.Interface, error)
+	// TODO remove.  This should be rolled into `ClientSet`
+	KubernetesClientSetForVersion(requiredVersion *schema.GroupVersion) (kubernetes.Interface, error)
```

Comment:
Why add something that we don't want to have anymore?

Reply:
Thanks for the review, I have addressed the comment.